CHANGELOG.md

```diff
@@ -56,7 +57,7 @@ Terminal commands changed (#135)
 - \#182 Boolean simulation settings now also take affect
 - Demand is no longer aggregated across sectors when processing/plotting in E1 (#169)
 
-## [0.1.0] -2020-01-30
+## [0.1.1] -2020-01-30
```

Comment:
Is this the PR to change the version number in the changelog?

Reply:
*g* No.... no, it is not. Yes, I should have created a seperate PR. xD

Reply:
These small fixes could be even done in `dev` branch.